pkg/apis/config/v1alpha2/mesh_config.go

```diff
@@ -65,6 +65,15 @@ type SidecarSpec struct {
 
 	// Resources defines the compute resources for the sidecar.
 	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
+
+	// TLSMinimumProtocolVersion defines the minimum TLS protocol version that the sidecar supports
+	TLSMinimumProtocolVersion string `json:"tls_minimum_protocol_version,omitempty"`
```

Comment:
I don't think the min/max TLS versions or the service cert validity duration should be added to a new cert object since they are specific to the sidecars. It might make sense to include key size.